### PR TITLE
Negative sampling

### PR DESCRIPTION
**Pull request by kevinoriordan**
Content by Jason Tam. I'm just acting as secretary 🙂

### CODE REVIEW

File: _authors/jason-tam.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: Jason Tam
+job-title: Data Scientist
+mini-bio: > # 280 character limit! No html, no quotes, no line breaks, just one big string of 280 characters or less.
+  Jason is a Data Scientist focusing on delivering to HBC customers cutting edge web and mobile personalised experience. Since he joined HBC in 2016, the personalization team has delivered state of the art recommendation systems.
+
+# list one or multiple
+social-media-profiles:
+- platform: github
+  url: https://github.com/jtam
```
Comment (JasonTam) [inline _authors/jason-tam.html]: https://github.com/JasonTam

File: _authors/jason-tam.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  handle: jtam
```
Comment (JasonTam) [inline _authors/jason-tam.html]: JasonTam

File: _posts/2018-03-23-negative-sampling-in-numpy.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
+---
+title: Negative Sampling (in Numpy)
+description: Alright, time to have some fun exploring efficient negative sampling implementations in NumPy...
+author: Jason Tam
+
+date: '2018-03-23'
+
+categories:
+- data science
```
Comment (JasonTam) [inline _posts/2018-03-23-negative-sampling-in-numpy.md]: well, not really -- but I guess there's no better category :no_mouth:

File: _posts/2018-03-23-negative-sampling-in-numpy.md [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+#
+tags:
```
Comment (JasonTam) [inline _posts/2018-03-23-negative-sampling-in-numpy.md]: python, algorithms, ... interview questions (lol jk)

**Comment by JasonTam** [comment]
Could we just scrap the image? Or would it be better to have some random unrelated scenic landscape ? :grin:

**Comment by JasonTam** [comment]
https://ixionx.deviantart.com/art/Emo-Happy-Smiley-Wallpaper-128689337  

> License:
> Some rights reserved. This work is licensed under a 
> Creative Commons Attribution-Noncommercial-Share Alike 3.0 License.

**Comment by designfirst** [comment]
yeah, I suppose this is a difficult topic to illustrate, but not impossible. I would either scrap the image or have a conversation with a designer on our team to help you illustrate your topic.

**Pull request merged**